doc/languages-frameworks/beam.section.md

```diff
@@ -0,0 +1,116 @@
+# BEAM Languages (Erlang, Elixir & LFE)
+
+## Introduction
+
+In this document and related Nix expressions, we use the term, *BEAM*,
```

Comment:
It would be nice not to hard-wrap the lines. All modern editors (even stuff like nano) have soft-wrapping, while tools like `git diff` cannot handle hard-diffing.